modules/nw-egress-router-about.adoc

```diff
@@ -21,17 +29,30 @@ The egress router image is not compatible with Amazon AWS, Azure Cloud, or any o
 [id="nw-egress-router-about-modes_{context}"]
 == Egress router modes
 
-In _redirect mode_, an egress router pod sets up iptables rules to redirect traffic from its own IP address to one or more destination IP addresses. Client pods that need to use the reserved source IP address must be modified to connect to the egress router rather than connecting directly to the destination IP.
+In _redirect mode_, an egress router pod configures `iptables` rules to redirect traffic from its own IP address to one or more destination IP addresses. Client pods that need to use the reserved source IP address must be modified to connect to the egress router rather than connecting directly to the destination IP.
 
+ifdef::openshift-sdn[]
 In _HTTP proxy mode_, an egress router pod runs as an HTTP proxy on port `8080`. This mode only works for clients that are connecting to HTTP-based or HTTPS-based services, but usually requires fewer changes to the client pods to get them to work. Many programs can be told to use an HTTP proxy by setting an environment variable.
 
 In _DNS proxy mode_, an egress router pod runs as a DNS proxy for TCP-based services from its own IP address to one or more destination IP addresses. To make use of the reserved, source IP address, client pods must be modified to connect to the egress router pod rather than connecting directly to the destination IP address. This modification ensures that external destinations treat traffic as though it were coming from a known source.
 
 Redirect mode works for all services except for HTTP and HTTPS. For HTTP and HTTPS services, use HTTP proxy mode. For TCP-based services with IP addresses or domain names, use DNS proxy mode.
+endif::openshift-sdn[]
+
+ifdef::ovn[]
+[NOTE]
+====
+The egress router CNI plug-in supports _redirect mode_ only.
+This is a difference with the egress router that you can deploy with OpenShift SDN.
+Unlike the egress router for OpenShift SDN, the egress router CNI plug-in does
+not support _HTTP proxy mode_ or _DNS proxy mode_.
```

Comment:
But we aren't showing these modes anymore, because of the conditional, so the reader won't really understand what these missing modes actually are. Unless they've read the OpenShift SDN variant of the page.

I'd probably call it an _egress router implementation_, because technically we're deploying egress router _pods_, and there is no actual _egress router_ entity, except the overall implementation itself.

Reply:
+1. Makes sense, maybe we could just omit that part or refer to the legacy egress router documentation, whatever you feel better with!

Reply:
I spoke with Jason, but didn't record my rationale here...  My thought is that while the target audience is someone new--who might not know much about SDN, another audience is the expert OpenShift admin. If that expert admin is familiar with SDN, I didn't want that person to think the doc was incomplete because it failed to include DNS and HTTP.